build/regressions.config.yml

```diff
@@ -39,6 +39,29 @@ sources:
         - sebastienros
         - agocke
       template: build
+  - name: MVC 
+    table: MvcBenchmarks
+    rules:
+      - include: .
+      - include: Mvc
+        labels: 
+          - area-mvc
+      - exclude: ulib
+      - exclude: arm
+    daysToLoad: 14
+    stdevCount: 7
+    daysToSkip: 1 # allow one day of benchmarks before filing an issue, to detect false positives
+    regressions:
+      probes: 
+        - path: jobs.load.results['wrk/rps/mean']
```

Comment:
@sebastienros we are using bombardier.

Should this point to some other path? Do the scripts below need any adjustment?

Reply:
The probes is an array. In the table we have a mix of clients, wrk and bombardier, so we need to add a new line, like this:
```
- path: jobs.load.results['bombardier/rps/mean']
```

This just says that the measurements named "bombarder/rps/mean" need to be tracked. If we wanted to track latency we could add the name of the result here too.

Reply:
I've changed it to `- path: jobs.load.results['bombardier/rps/mean']`

Reply:
I fixed it, please take a look